Hazel/src/Platform/OpenGL/OpenGLVertexArray.h

```diff
@@ -8,16 +8,16 @@ namespace Hazel {
 	{
 	public:
 		OpenGLVertexArray();
-		virtual ~OpenGLVertexArray();
+		~OpenGLVertexArray() override;
 
-		virtual void Bind() const override;
```

Comment:
Why get rid of `virtual`? It is much more visually distinct then just the `override`.

I'm in favor of keeping both `virtual` and `override`, especially in cases where there isn't existing conventions to identify what is part of an implemented interface or not.